scripts/base/protocols/conn/main.zeek

```diff
@@ -158,6 +158,11 @@ export {
 		## *uid* values for any encapsulating parent connections
 		## used over the lifetime of this inner connection.
 		tunnel_parents: set[string] &log &optional;
+		## For IP-based connections, this contains the protocol
+		## identifier passed in the IP header. This is different
+		## from the ``proto`` field in that this value comes
```

Comment:
Nit: to render right this has to be `*proto*`, ReST style. (See e.g. `conn_state` higher up.)